packages/@n8n/eslint-config/package.json

```diff
@@ -2,32 +2,58 @@
   "name": "@n8n/eslint-config",
   "private": true,
   "version": "0.0.1",
+  "type": "module",
   "exports": {
-    "./base": "./base.js",
-    "./frontend": "./frontend.js",
-    "./local-rules": "./local-rules.js",
-    "./node": "./node.js",
-    "./shared": "./shared.js"
+    "./base": {
+      "types": "./dist/configs/base.d.js",
+      "default": "./dist/configs/base.js"
+    },
+    "./frontend": {
+      "types": "./dist/configs/frontend.d.js",
+      "default": "./dist/configs/frontend.js"
+    },
+    "./node": {
+      "types": "./dist/configs/node.d.js",
+      "default": "./dist/configs/node.js"
+    }
+  },
+  "dependencies": {
```

Comment:
Why are some regular deps and others dev deps?

Reply:
Moved all to dev dependencies, I think that makes sense for this package